tasks/65_smaller-common-multiple/smaller-common-multiple.js

```diff
@@ -0,0 +1,15 @@
+export function smallestCommons(arr) {
+  arr = arr[0] < arr[1]? arr : [...arr].reverse();
```

Comment:
Please take a look at this https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/